packages/unlock-js/src/web3Service.ts

```diff
@@ -509,4 +509,18 @@ export default class Web3Service extends UnlockService {
     const id = await lockContract.tokenOfOwnerByIndex(owner, index)
     return id.toNumber()
   }
+
+  /**
+   * Returns the number of keys already sold
+   * @param lockAddress
+   * @param network
+   * @returns
+   */
+  async totalSupply(lockAddress: string, network: number) {
```

Comment:
I think we have that thru `getLock` already... but maybe it is worth adding independently?

Reply:
Yes, I see we have it also inside `getLock`, there is some place we need only this is information, we can avoid to use `getLock` in those scenarios